solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -878,33 +973,60 @@ describe("RandomBeacon - Relay", () => {
           expect(await isGroupTerminated(0)).to.be.equal(true)
         })
 
-        it("should call staking contract to seize the min stake", async () => {
-          await expect(tx)
-            .to.emit(staking, "Seized")
+        it("should reward the notifier", async () => {
+          await expect(reportTx)
+            .to.emit(staking, "NotifierRewarded")
             .withArgs(
-              to1e18(100000),
-              params.unauthorizedSigningNotificationRewardMultiplier,
               notifier.address,
-              membersAddresses
+              constants.tokenStakingNotificationReward
+                .mul(params.unauthorizedSigningNotificationRewardMultiplier)
+                .div(100)
+                .mul(membersIDs.length)
             )
         })
 
+        it("should slash the minimum stake for all group members", async () => {
+          for (let i = 0; i < membersAddresses.length; i++) {
+            const stakingProvider =
+              await randomBeacon.operatorToStakingProvider(membersAddresses[i])
+
+            await expect(slashingTx)
+              .to.emit(staking, "TokensSeized")
+              .withArgs(
+                stakingProvider,
+                params.unauthorizedSigningSlashingAmount,
+                false
+              )
+          }
+        })
+
         it("should emit unauthorized signing slashing event", async () => {
-          await expect(tx)
+          await expect(reportTx)
             .to.emit(randomBeacon, "UnauthorizedSigningSlashed")
-            .withArgs(0, to1e18(100000), membersAddresses)
+            .withArgs(
+              0,
+              params.unauthorizedSigningSlashingAmount,
+              membersAddresses
+            )
+        })
+
+        it("should not emit UnauthorizedSigningSlashingFailed", async () => {
+          await expect(reportTx).to.not.emit(
+            randomBeacon,
+            "UnauthorizedSigningSlashingFailed"
+          )
         })
       })
 
-      context("when token staking seize call fails", async () => {
-        let tokenStakingFake: FakeContract<IRandomBeaconStaking>
+      context.skip("when token staking seize call fails", async () => {
```

Comment:
Can we add `// FIXME: Blocked by https://github.com/defi-wonderland/smock/issues/101`?

Reply:
f931643f8c9330c677ed937b457aed7e45072fe9